build.rs

```diff
@@ -172,11 +172,10 @@ async fn main() -> anyhow::Result<()> {
     println!("cargo:rerun-if-env-changed=WASI_PREVIEW1_COMMAND_COMPONENT_ADAPTER");
     println!("cargo:rerun-if-env-changed=WASI_PREVIEW1_REACTOR_COMPONENT_ADAPTER");
 
-    #[cfg(feature = "build-adapters")]
-    {
-        let out_dir = env::var("OUT_DIR")
-            .map(PathBuf::from)
-            .context("failed to lookup `OUT_DIR`")?;
+    let out_dir = env::var("OUT_DIR")
+        .map(PathBuf::from)
+        .context("failed to lookup `OUT_DIR`")?;
+    if cfg!(feature = "docs") {
```

Comment:
Should this condition/the blocks be inverted?

Reply:
yes :eyes: